src/stores/store.ts

```diff
@@ -8,9 +8,11 @@ import {
 import { setupListeners } from "@reduxjs/toolkit/dist/query";
 import { TypedUseSelectorHook, useDispatch, useSelector } from "react-redux";
 
-import { balanceApi } from "./api/balanceApi";
+import assetBalanceReducer from "./slices/assetBalanceSlice";
+import collectiblesReducer from "./slices/collectiblesSlice";
```

Comment:
It is how `redux` organises it's state. You can store global state in so called slices. We could in theory also put it all into one big slice but like this it is more maintainable and readable.

And Redux is used to store global state. This is to avoid having to drill down the whole app's state through props into all child components who need it.
For instance the balances are needed by different components:
- The component that generates the "Drain" CSV content
- The donation dialog to offer tokens from the user's balance
- The Editor parser to check if the user's balance is high enough for the transfers
- ... 

That's why we store it in a global redux state so that all those components can easily retrieve it through selectors.